Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG GOLANG_BUILDER=golang:1.19
-ARG OPERATOR_BASE_IMAGE=gcr.io/distroless/static:nonroot
+ARG GOLANG_BUILDER=registry.access.redhat.com/ubi9/go-toolset:1.19
+ARG OPERATOR_BASE_IMAGE=registry.access.redhat.com/ubi9/ubi-minimal:latest
```

Comment:
*if* we decide to not go with FIPS also on upstream (which is the default passing those values to `GO_BUILD_EXTRA_ARGS` and `GO_BUILD_EXTRA_ENV_VARGS`), otherwise you can't, because `CGO_ENABLED=1` make the binary dynamically linked and it's a requirement (as stated in `payload-check` code).

Reply:
FYI https://github.com/openstack-k8s-operators/docs/blob/main/fips.md